be/src/exec/json_scanner.cpp

```diff
@@ -609,6 +614,39 @@ Status JsonReader::_construct_row_without_jsonpath(simdjson::ondemand::object* r
     }
     return Status::OK();
 }
+// process debezium-json record
+Status JsonReader::_construct_row_by_debezium(simdjson::ondemand::object* row, Chunk* chunk) {
+    if (UNLIKELY(_op_col_index < 0)) {
+        return Status::InternalError("Debezium json must apply to PRIMARY KEY table.");
```

Comment:
> 1. Is it correct to use `_op_col_index` to judge?
> 2. It's better to show the parameters. The message could be: `Debezium ..., current table type is $1` if it's possible.

1. It is correct because `debezium-json` parser is only useful for a `PRIMARY KEY` table with `_op_col`.
2. Ok, I will modify it.